common4j/src/main/com/microsoft/identity/common/java/platform/JweResponse.java

```diff
@@ -89,9 +91,13 @@ public String getAuthenticationTag() {
     }
 
     public static JweResponse parseJwe(String jwe) throws JSONException {
+        final Span span = Span.current();
         JweResponse response = new JweResponse();
 
         String[] split = jwe.split("\\.");
+
+        span.setAttribute(AttributeName.jwt_valid.name(), split.length >= 4);
```

Comment:
probably should turn 4 into a constant, with comment (on why 4 specifically).

Reply:
This was based on the existing code. Converted to constant and added comment: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1847/commits/d07cc33731eefe48069d5beac31dad4b0c71c22a